components/wmtk_components/embedded_component/internal/EmbeddedRemeshing.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <igl/adjacency_list.h>
+#include <igl/edges.h>
+#include <igl/triangle/triangulate.h>
+#include <wmtk/Scheduler.hpp>
+#include <wmtk/TriMesh.hpp>
+
+namespace wmtk::components::internal {
+
+class EmbeddedRemeshing2D
+{
+public:
+    EmbeddedRemeshing2D(
+        Eigen::MatrixXi& E_,
+        Eigen::MatrixXd& V_,
+        double blank_rate_ = 0.5,
+        double resolute_area = 0.1);
+    void compute_bounding_vaule(double& max_x, double& max_y, double& min_x, double& min_y);
+    void process();
+
+    // can also be int, double any type, just code as bool for now.
+    Eigen::MatrixXi E;
```

Comment:
We decided to use `m_...` as convention for member variables, so in that case it should be `m_E`. Honestly, I don't like the libigl notation with single capital letters. I would prefer, if you just call them `m_edges`, `m_vertices`, and so on.